content/docs/user-guide/merge-conflicts.md

```diff
@@ -0,0 +1,198 @@
+# Merge Conflicts
+
+Sometimes multiple members of a team might want to edit the same file/directory
+that is tracked by DVC. And when the time comes to merge their changes together
+one will run into a merge conflict.
+
+## dvc.yaml
+
+This one is no different from your regular merge conflicts in code.
+
+```
+stages:
+  prepare:
+    cmd: python src/prepare.py data/data.xml
+    deps:
+<<<<<<< HEAD
+    - data/big.xml
+=======
+    - data/small.xml
+>>>>>>> branch
+    - src/prepare.py
+    params:
+    - prepare.seed
+    - prepare.split
+    outs:
+    - data/prepared
+```
+
+See https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging for
+more details.
+
+## dvc.lock
+
+You can safely delete this file and then just run `dvc repro` after the merge is
+done.
+
+```
+prepare:
+  cmd: python src/prepare.py data/data.xml
+  deps:
+<<<<<<< HEAD
```

Comment:
repro or commit?

Maybe no YAML sample is needed here since the rec is to delete it and regenerate it.

Reply:
Better to repro. Commit might have unexpected results.

Reply:
Oh I missed this one. OK but maybe we should mention both options?

Reply:
Yeah I think if there's more than one option they should at least be briefly mentioned.

Reply:
We could mention commit but only as something that people shouldn't really use here because it might bring unexpected results.

Reply:
But I would handle that separately later. This doc is about _the proper_ way to do things.

Reply:
OK, I'll remove the mention. I don't understand what unexpected results you're referring to though.

Reply:
@jorgeorpinel It will commit whatever trash you might have in your output, which might not be what you want.

Reply:
Gotcha. I guess the data in the machine that os merging would correspond to the `HEAD` version so `commit` only works if you decide to keep that version. So you're right, not worth mentioning that case, probably. Except that commit will be much faster and efficient than repro in that case...